volume/volume.go

```diff
@@ -219,6 +219,9 @@ type FilesystemTrimDriver interface {
 	// AutoFilesystemTrimStatus returns the status of auto fs trim
 	// operations on volumes
 	AutoFilesystemTrimStatus(request *api.SdkAutoFSTrimStatusRequest) (*api.SdkAutoFSTrimStatusResponse, error)
+	// AutoFilesystemTrimUsage returns the usage of auto fs trim
+	// operations on locally mounted volumes
```

Comment:
nit
try rephrasing to something like
" AutoFilesystemTrimUsage returns the volume usage and trimmable space of locally mounted pxd volumes"